M2Loader.js

```diff
@@ -2911,23 +2911,33 @@ class SequenceManager {
 
 			if ( animation.variationIndex === variationIndex ) {
 
-				const mixer = this._mixers.get( animation.root );
-				const action = mixer.clipAction( animation.clip );
-				action.play();
+				if ( animation.flags & M2_SEQUENCE_EMBEDDED_DATA ) {
+
+					const mixer = this._mixers.get( animation.root );
+					const action = mixer.clipAction( animation.clip );
+					action.play();
+
+				} else {
+
+					// TODO: Add support for sequences with external animation data (.anim files)
+
+					console.warn( 'THREE.M2Loader: Sequences with external animation data not yet supported.' );
```

Comment:
For simplicity, I have moved the `M2_SEQUENCE_EMBEDDED_DATA` check to `playSequence()`. Meaning the manager now reports all sequences and variations again but warns the users if animation data are not available.